samples/hello-slick/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.18
+sbt.version=1.3.8
```

Comment:
above you went to 1.4.5, is it just an oversight this isn't the same?

Reply:
ditto on the other samples

Reply:
oversight

Reply:
changed to 1.4.7